packages/os/storewolf.service

```diff
@@ -9,7 +9,7 @@ RefuseManualStop=true
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/storewolf --data-store-base-path /var/lib/bottlerocket/datastore
+ExecStart=/usr/bin/storewolf --data-store-base-path /var/lib/bottlerocket/datastore --inventory-file-symlink-path /var/lib/bottlerocket/inventory/application.json
```

Comment:
Does a new symlink on the system need to be deleted on downgrade? @bcressey? Seems harmless, right?

Reply:
I'm not sure - my assumption is "probably" since a downgrade would likely carry a different version of the rpm json inventory.  But with the "delete" functionality that will remove an existing symlink if it's there is abit of a moot point.